GitUpKit/Utilities/GIViewController.m

```diff
@@ -341,6 +360,10 @@ - (void)viewDidFinishLiveResize {
   ;
 }
 
+- (void)viewDidChangeEffectiveAppearance {
+  ;
```

Comment:
A comment might be nice here to explain why this is being overridden.

Reply:
Can do. Was just following convention in the file.

Reply:
Addressed in next push